docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md

```diff
@@ -1,211 +1,172 @@
-A predefined node called `RectangleArena` is designed to accomplish this task quickly.
-
-> **Hands on**: Select the last node of the Scene Tree view ([Background](../reference/background.md)).
-Click on the `Add` button at the top of the Scene Tree view.
-In the open dialog box, choose `PROTO (Webots) / objects / floors / RectangleArena`.
-The new node has been added and appears far away.
-Use the left click and the wheel of the mouse in the 3D view to choose a better viewpoint.
-
-However, the rectangle arena appears black because the scene is still unlit.
-Now we would like to add some light to the scene.
-
-> **Hands on**: Select the last node of the Scene Tree view (`RectangleArena`).
-Click on the `Add` button.
-In the open dialog box, choose `New node / DirectionalLight`.
-The new node has been added and we can admire our rectangle arena's colors.
-
-It's a good time to improve the scene light.
-
-> **Hands on**: Modify the following fields of the `DirectionalLight` node:
-
-    - `intensity` to 2.3.
-    - `direction` to [-0.33 -1 0.5].
-    - `castShadows` to TRUE.
-
-<!-- -->
-
-> **Note**: In the Scene Tree view, the fields are displayed in a different color (depending on the theme) if they differ from their default values.
-
-Now your environment should look like the one depicted in the [figure](prerequisites.md#the-webots-main-window-splits-into-four-dockable-subwindows-the-scene-tree-view-on-the-left-hand-side-including-a-panel-at-the-bottom-for-editing-fields-values-the-3d-view-in-the-center-the-text-editor-on-the-right-hand-side-and-the-console-at-bottom-of-the-window-note-that-some-of-these-subwindows-have-a-toolbar-with-buttons-the-main-menus-appear-on-the-top-of-the-main-window-the-virtual-time-counter-and-the-speedometer-are-displayed-in-the-right-part-of-the-main-toolbar-the-status-text-is-displayed-in-the-bottom-left-of-the-main-window).
-
-> **Hands on**: Save the new world into your project by selecting the `File / Save World As...` menu item.
-Using the dialog box save the world into the "my\_webots\_projects/tutorials/worlds/my\_first\_simulation.wbt" file location.
-
-<!-- -->
+### Create a New World
 
-> **Hands on**: Reload the simulation by selecting the `File / Reload World` menu item.
+A **World** is a file containing information like where the objects are, what they look like, how they interact with each other, what is the color of the sky, how is defined the gravity, friction, masses of the objects, etc.
+It defines the initial state of a simulation.
+The different objects are called **Nodes** and are organized hierarchically in a **Scene Tree**.
+Therefore, a node may contain sub-nodes.
+A world is stored in a file having the `.wbt` extension.
+The file format is derived from the **VRML97** language, and is human readable.
+The world files must be stored directly in a directory called `worlds`.
 
-<!-- -->
+> **Hands on #2**: Pause the current simulation by clicking on the `Pause` button of the 3D view.
```

Comment:
We should link to https://cyberbotics.com/doc/guide/the-user-interface?version=fix-tutorial-1#simulation-menu